apps/api/src/app/inversify.config.ts

```diff
@@ -47,6 +42,20 @@ import {
   usdServiceSymbol,
 } from '@cowprotocol/services';
 import ms from 'ms';
+import pino from 'pino';
+
+const DEFAULT_CACHE_VALUE_SECONDS = ms('2min') / 1000; // 2min cache time by default for values
+const DEFAULT_CACHE_NULL_SECONDS = ms('30min') / 1000; // 30min cache time by default for NULL values (when the repository isn't known)
+
+const CACHE_TOKEN_INFO_SECONDS = ms('24h') / 1000; // 24h
+
+// Configure the logger
+const logger = pino({
+  level: process.env.LOG_LEVEL || 'info',
+  formatters: {
+    level: (label) => ({ level: label }),
+  },
+});
```

Comment:
@yvesfracari why do we add the log config here, shouldn't we use the same one as https://github.com/cowprotocol/bff/blob/main/apps/api/src/main.ts#L5 ?

Reply:
I address it, cause I want to deploy a new release of bff with some fixes for the logging and I want to fix this small detail. Let me know if it makes sense